tests/em/fdem/forward/test_FDEM_forward.py

```diff
@@ -17,6 +19,42 @@
 SrcList = ['RawVec', 'MagDipole_Bfield', 'MagDipole', 'CircularLoop']
 
 
+class SrcLocTest(unittest.TestCase):
+    def test_src(self):
```

Comment:
and here is a test to make sure that the user can input 1D or 2D arrays (and that incorrect shapes will be caught)

Reply:
that's great!